RELEASE_NOTES

```diff
@@ -3,8 +3,60 @@ For more detailed information, please see the git log.
+v2.9.0 (September 23rd 2016)
+----------------------------
+
+feature release
+- added example easyconfig files for 88 new software packages:
+    ADMIXTURE (#3359), angsd (#3593), ASHS (#3429), AutoDock (#3465), AutoGrid (#3466), BayeScan (#2748, #3356),
+    BayPass (#3451), Bazel (#3379), Blender (#3553, #3558), bwakit (#3567), BXH_XCEDE_TOOLS (#3410), CastXML (#3403),
+    CHASE (#3304), configparser (#3368, #3424), configurable-http-proxy (#3380), cppcheck (#3508), CRPropa (#779),
+    DicomBrowser (#3432), DMTCP (#3422), entrypoints (#3368, #3424), f90cache (#3570), fastPHASE (#3343),
+    fastQValidator (#3192), FFindex (#1135), FragGeneScan (#1198), gdc-client (#3399), gflags (#3417), glog (#3417),
+    GRIT (#3561), H5hut (#3431), HAPGEN2 (#3344), HEALPix (#779), IMOD (#1187, #3347), IronPython (#607),
+    jhbuild (#3476), jupyterhub (#3380), Keras (#3581), khmer (#1158), LeadIT (#3345, #3599), LevelDB (#3417),
+    libbitmask (#3481), libcpuset (#3481), LMDB (#3417), log4cplus (#1136), MACH (#3346), Mako (#3460), Maq (#3428),
+    MetaGeneAnnotator (#3307), Metal (#3324), Mono (#607), MyMediaLite (#607), nco (#2575), nose-parameterized (#3579),
+    OpenEXR (#3553), OpenImageIO (#3553), path.py (#3368, #3424), PCRE2 (#3325), pftoolsV3 (#3317), PHASE (#3385),
+    PLAST (#3288), PLINKSEQ (#3402), POV-Ray (#3551), ProbABEL (#3108), prompt-toolkit (#3368, #3424),
+    protobuf-python (#3563), PSORTb (#3317), py (#3403, #3482), pygccxml (#3403, #3482), pyGIMLi (#3403, #3482),
+    pyplusplus (#3403, #3482), PyQt5 (#3533), Pyro4 (#3527), pytest (#3403, #3482), QCA (#3595), RDMC (#1137),
+    S.A.G.E. (#3427), SDL2 (#3551), SHORE (#3531), SimVascular (#3555), SortMeRNA (#3326), SUMACLUST (#3316),
+    SUMATRA (#3316), Text-CSV (#3323), Triangle (#3403), VEGAS (#3457), VirSorter (#3307), wcwidth (#3368, #3424),
+    X11 (#3340)
+- added new easyconfigs for existing toolchains:
+    - CrayGNU + CrayIntel 2016.06 (#3377)
+    - foss 2016.07 (#3517) + 2016.09 (#3523),
+    - iomkl 2016.07 (#3458)
+    - pomkl 2016.09 (#3516)
+- added additional easyconfigs for various supported software packages, including:
+    FFTW 3.3.5, GCC 4.9.4 + 6.2.0, GROMACS 5.1.4, IPython 5.1.0, LLVM 3.9.0, Mesa 12.0.1, OpenCV 3.1.0, OpenFOAM 4.0,
+    OpenMPI 2.0.1, ParaView 5.1.2, PGI 16.7, QuantumESPRESSO 5.4.0, Qt5 5.7.0, R-bundle-Bioconductor 3.3, VTK 7.0.0,
+    Yade 2016.06a
+- various enhancements, including:
+    - adjust PSI4 easyconfigs for updated easyblock (#3312)
+    - clean up libxml2 easyconfigs according to updated libxml2 easyblock (#3479, #3509)
+    - significantly speed up verifying of dumped easyconfig by resorting to 'shallow' parsing (#3520)
+    - include sanity checks for all MATIO config files (#3528)
+    - remove --with-tcl-config/--with-tk-config from R easyconfig, already done in R easyblock (#3580)
+- various bug fixes, including:
+    - disable testing in all ParaView 4.4.0 easyconfigs, required download is too much of a PITA (#3178)
```

Comment:
1 idea, for the future intern:
* make all these difficult tests being able to invoke with some special parameter, fi. --pedantic-testing